### DISCUSSION TITLE
git-lfs.exe writing to C:\$LogFile (NTFS Volume Log) @ 100% disk usage

### DISCUSSION THREAD

**Discussion by EnviralDesign** [git-lfs/git-lfs#4743] [Q&A]
I noticed today when pushing a commit with 6 large binary files to a repo, that my upload speed was capped at around 60 Mbps, which dropped as some of the smaller files finished. It equated roughly to 10Mpbs per file that was uploading.

While all 6 were in the process of uploading, my m2 ssd where the repo lives was showing 100% disk write usage in task manager. This dropped by increments as certain files finished uploading.

In windows Resource Monitor, the top line item in disk write speed was a line that said 
```
git-lfs.exe writing to C:\$LogFile
```

Does anyone know what this is about?

**Comment by bk2204** [comment]
Hey,

The `$LogFile` path is the NTFS file system journal.  Whenever a write occurs to your disk, metadata is written first to the journal to ensure the file system remains in a consistent state after a crash.

Seeing high disk usage there probably means that something is writing a lot.  If you're doing a push, that probably isn't Git LFS itself, but it's possible if you have an antivirus running, very especially one other than Windows Defender, then it may inject code into the process which causes this to occur.  Various other monitoring tools can also cause this for the same reason.

If you were doing a fetch or checkout, then it would be normal to see a lot of disk usage in that case, but not so much for a push.

**Reply by EnviralDesign** [reply]
Hey thanks a lot @bk2204 the tip about other monitoring I'll look into. I don't have another AV but it might be another running process getting in the way.

**Reply by gonchik** [reply]
@EnviralDesign did you figure out that problem ? 
I see significant slowness as well. 

<img width="1157" alt="image" src="https://github.com/git-lfs/git-lfs/assets/12548235/be073a73-063d-4db4-8dad-5e4b87d4104e">
